src/plugins/data/public/ui/dataset_selector/configurator.tsx

```diff
@@ -41,8 +41,9 @@ export const Configurator = ({
   const languages = type?.supportedLanguages(baseDataset) || [];
 
   const [dataset, setDataset] = useState<Dataset>(baseDataset);
-  const [timeFields, setTimeFields] = useState<DatasetField[]>();
+  const [timeFields, setTimeFields] = useState<DatasetField[]>([]);
   const [timeFieldName, setTimeFieldName] = useState<string | undefined>(dataset.timeFieldName);
+  const noTimeFilter = "I don't want to use the time filter";
```

Comment:
this should have the translation id in the create index pattern page 
![image](https://github.com/user-attachments/assets/56d319f9-433e-4dea-89c7-bff1b8eed082)